Paintroid/src/main/java/org/catrobat/paintroid/eventlistener/ChangeActiveLayerEventListener.java

```diff
@@ -0,0 +1,10 @@
+package org.catrobat.paintroid.eventlistener;
+
+import org.catrobat.paintroid.tools.Layer;
+
+/**
+ * Created by dzombeast on 13.02.2016.
+ */
+public interface ChangeActiveLayerEventListener {
```

Comment:
This class name is not very inspiring....
maybe "ActiveLayerChangeListener" or "OnActiveLayerChangedListener"